src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+@import 'variables';
```

Comment:
No need to import it here if you imported in index.scss (but before `@import 'card'`)